runtime/development/src/lib.rs

```diff
@@ -1571,22 +1570,6 @@ impl orml_asset_registry::Config for Runtime {
 	type WeightInfo = ();
 }
 
-pub struct DummyOutboundQueue;
```

Comment:
This shouldn't be part of latest main, maybe a re-base might fix this?

Reply:
I am on latest main. I hope at least ^^